sdk/search/azure-search-documents/azure/search/documents/_service/aio/_search_service_client_async.py

```diff
@@ -45,6 +45,14 @@ class SearchServiceClient(HeadersMixin):  # pylint: disable=too-many-public-meth
     def __init__(self, endpoint, credential, **kwargs):
         # type: (str, AzureKeyCredential, **Any) -> None
 
+        try:
```

Comment:
It's a shame we have to duplicate this code, should we have a SeachServiceClientBase?

Reply:
I was think about the same - Will use this PR to do that